scripts/e2etests/RunFunctionalTests.ps1

```diff
@@ -6,28 +6,23 @@ param (
     [Parameter(Mandatory=$true)]
     [int]$EachTestTimoutInSecs,
     [Parameter(Mandatory=$true)]
-    [string]$NuGetDropPath,
-    [Parameter(Mandatory=$true)]
-    [string]$FuncTestRoot,
-    [Parameter(Mandatory=$true)]
-    [string]$RunCounter,
-    [Parameter(Mandatory=$true)]
-    [ValidateSet("17.0")]
-    [string]$VSVersion)
+    [string]$FuncTestRoot)
```

Comment:
We should probably port a version of these changes to release-5.11.x right?

Reply:
Yes, I think we'll need to, in order to get the servicing branch to run tests successfully while dev16 and dev17 are installed side-by-side on the CI agents.